roofit/doc/developers/roofit_ad.md

```diff
@@ -865,24 +927,25 @@ console (useful for debugging).
 from the provided data and prepares them for evaluation.
 
   - Input: `funcName` (function name), `head` (structure of the function),
-`paramSet` (input function's parameters), `data` (optional data points).
-  - Output: Parameters, observables and other related information (e.g.,
-output size of the provided function).
+    `paramSet` (input function's parameters), `data` (optional data points).
+  - Output: Parameters, observables and other related information (e.g., output
+    size of the provided function).
 
 
 
 [^1]: For more details, please see this paper on [Automatic Differentiation of
-Binned Likelihoods With Roofit and Clad]
+Binned Likelihoods with Roofit and Clad]
 
-[Automatic Differentiation of Binned Likelihoods With Roofit and Clad]: https://arxiv.org/abs/2304.02650
+[Automatic Differentiation of Binned Likelihoods with Roofit and Clad]: https://arxiv.org/abs/2304.02650
 
-[^2]: For more details, please see this paper on [GPU Accelerated Automatic Differentiation With Clad]
+[^2]: For more details, please see this paper on [GPU Accelerated Automatic Differentiation with Clad]
 
-[GPU Accelerated Automatic Differentiation With Clad]: https://arxiv.org/abs/2203.06139
+[GPU Accelerated Automatic Differentiation with Clad]: https://arxiv.org/abs/2203.06139
 
 [RooFit]: https://root.cern/manual/roofit/
 
 [Clad]: https://compiler-research.org/clad/
 
 [EvaluateFuncs]: https://github.com/root-project/root/blob/4e8c577dfd6a19d7c38a74e3074b406a598bf76a/roofit/roofitcore/inc/RooFit/Detail/EvaluateFuncs.h
 
+[AnalyticalIntegrals]: https://github.com/root-project/root/blob/4e8c577dfd6a19d7c38a74e3074b406a598bf76a/roofit/roofitcore/inc/RooFit/Detail/AnalyticalIntegrals.h
```

Comment:
```suggestion
[AnalyticalIntegrals]: https://github.com/root-project/root/blob/master/roofit/roofitcore/inc/RooFit/Detail/AnalyticalIntegrals.h
```
Instead of pointing to a random commit, can you maybe point to ROOT master? Also for all the other `root-project/root` links in this file.